ginga/gtk3w/Widgets.py

```diff
@@ -196,6 +216,12 @@ def set_enabled(self, tf):
         super(TextEntrySet, self).set_enabled(tf)
         self.entry.set_sensitive(tf)
 
+
+class GrowingTextEdit(WidgetBase):
+    def __init__(self, *args, **kwargs):
+        raise NotImplementedError
+
```

Comment:
Okay, maybe the `NotImplementedError` additions are overkill. Should I just remove them?